src/main/resources/db/changelog/changes/009-insert-book-category-table.yaml

```
@@ -0,0 +1,50 @@
+              - column:
+                  name: category_id
+                  value: 2
+        - insert:
+            tableName: book_category
+            columns:
+              - column:
+                  name: book_id
+                  value: 3
+              - column:
+                  name: category_id
+                  value: 1
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)